FreeAPS/Sources/Localizations/Main/en.lproj/Localizable.strings

```diff
@@ -1863,7 +1863,7 @@ Enact a temp Basal or a temp target */
 "Enable UAM" = "Enable UAM";
 
 /* "Enable UAM" */
-"With this option enabled, the SMB algorithm can recognize unannounced meals. This is helpful, if you forget to tell Trio about your carbs or estimate your carbs wrong and the amount of entered carbs is wrong or if a meal with lots of fat and protein has a longer duration than expected. Without any carb entry, UAM can recognize fast glucose increasments caused by carbs, adrenaline, etc, and tries to adjust it with SMBs. This also works the opposite way: if there is a fast glucose decreasement, it can stop SMBs earlier." = "With this option enabled, the SMB algorithm can recognize unannounced meals. This is helpful, if you forget to tell Trio about your carbs or estimate your carbs wrong and the amount of entered carbs is wrong or if a meal with lots of fat and protein has a longer duration than expected. Without any carb entry, UAM can recognize fast glucose increasments caused by carbs, adrenaline, etc, and tries to adjust it with SMBs. This also works the opposite way: if there is a fast glucose decreasement, it can stop SMBs earlier.";
+"With this option enabled, the SMB algorithm can recognize unannounced meals. This is helpful, if you forget to announce your carbs or incorrectly estimate your carbs, or if a meal has a lot of fats and protein that delays carb absorption. Without any carb entry, UAM can recognize fast glucose rises caused by carbs, adrenaline, etc, and attempts to adjust it with SMBs. This also works the opposite way: if there is a fast glucose drop, it can stop SMBs earlier." = "With this option enabled, the SMB algorithm can recognize unannounced meals. This is helpful, if you forget to announce your carbs or incorrectly estimate your carbs, or if a meal has a lot of fats and protein that delays carb absorption. Without any carb entry, UAM can recognize fast glucose rises caused by carbs, adrenaline, etc, and attempts to adjust it with SMBs. This also works the opposite way: if there is a fast glucose drop, it can stop SMBs earlier.";
```

Comment:
With this option enabled, your pump can recognize unannounced meals. This is helpful, if you forget to announce your carbs or incorrectly estimate your carbs, if a meal has a lot of fats and protein that delays carb absorption. Without any carb entry, UAM can recognize fast glucose rises caused by carbs, adrenaline, etc, and attempts to adjust it. This also works the opposite way: if there is a fast glucose drop, it can stop insulin early.

With this option enabled, the algorithm can recognize unannounced meals. This is helpful, if you forget to announce your carbs or incorrectly estimate your carbs, or if a meal has a lot of fats and protein that delays carb absorption. Without any carb entry, UAM can recognize fast glucose rises caused by carbs, adrenaline, etc, and attempts to adjust it with SMBs (or TBRs if SMBs are not enabled). This also works the opposite way: if there is a fast glucose drop, it can stop SMBs earlier.